torch/_inductor/codegen/cpp_utils.py

```diff
@@ -448,6 +452,38 @@ def add_local_buffer(
         if global_buffer:
             self.local_to_global[local_buffer.get_name()] = global_buffer
             V.graph.removed_buffers.add(global_buffer.get_name())
+            # Patch the attr of SchedulerNode's outputs and outputs_by_name.
+            # Since the name of local buffer has not been added into the outputs
+            # of schedulerNode, which fails method of CSEProxy._update_store_cache
```

Comment:
Could you just add `if name in V.graph.buffers` to `_update_store_cache`?

Reply:
Sure, thanks for the suggestion and changed.